### PR TITLE
Fix indentation everywhere else

### PR DESCRIPTION
**Pull request by Heinrich-XIAO**
Indentation didn't seem to be fixed in a lot of places, so this pr should fix that.

### CODE REVIEW

**Comment by Naviary2** [comment]
Interesting, so it missed some TOMLs? There should be an automatic way to fix all indentation issues, I don't want you to spend all the time to manually fix each line. I suppose you could also search for every group of 4 spaces and mass-replace that way.

**Comment by Heinrich-XIAO** [comment]
> I suppose you could also search for every group of 4 spaces and mass-replace that way.

Some of the files won't work. like license being the simplest to fix. i think it needs some regex or something

**Comment by Naviary2** [comment]
Some files we could probably ignore, like ones that will never be modified. For example, I skipped modifying gl-matrix, which was a resource downloaded from online

**Comment by Heinrich-XIAO** [comment]
Why is #258 closed if this pr isn't?

**Comment by Naviary2** [comment]
There was just a few straggler files that still contained spaces for indentation, right? Is this one ready for review?

**Comment by Heinrich-XIAO** [comment]
I think so. I've started using the draft and open pr statuses correctly.

File: translation/changes.json [hunk 1]
```diff
@@ -1,145 +1,146 @@
...
+	"17": {
+		"note": "Added credits.language_heading and credits.language_credits, line 82",
+		"changes": []
+	},
+	"16": {
+		"note": "Deleted all news posts. The only keys you keep in the `news` section is `title` and `more_dev_logs`! Please verify your language's news posts have been translated correctly within translation/news!",
+		"changes": []
+	},
+	"15": {
+		"note": "Added news.sept9-2024, line 493",
```
Comment (Naviary2) [inline translation/changes.json:11]: Could you revert this?

**Comment by Naviary2** [comment]
Thanks!

**Pull request merged**